### PR TITLE
Fix 404

### PR DESCRIPTION
**Pull request by PencilNavigator**
### Type
- [x] Rewrite already written documentation.
- [ ] Add/remove new pages.

### Questions
- [x] Did you preview on your local machine beforehand?
- [x] Did you follow our [commit handbook](https://github.com/Atlas-OS/docs/blob/master/.github/CONTRIBUTING.md#commit-message)?

### Describe your pull request

- Fix incorrect hyperlink causing 404 issue (issue https://github.com/Atlas-OS/docs/issues/71)

- Correct `environment` spelling

### CODE REVIEW

**Review by Xyueta — APPROVED** [review]

File: docs/troubleshooting/common-issues/reticulating-splines.md [hunk 1]
```diff
@@ -2,16 +2,15 @@ After running the Playbook with the AME Wizard some people get stuck at the "Ret
...
 ### Fix 1
-Switch to a different network enviornment. (This includes but are not limited to using hotspot, using a different ISP, and using a VPN that proxies CMD)
+Switch to a different network environment. (This includes but are not limited to using hotspot, using a different ISP, and using a VPN that proxies system processes.)
 
 ### Fix 2
 1. Open Task Manager.
-2. Kill The curl executable,
-3. Kill powershell,
-4. Kill Windows Command Processor,
+2. Kill The `curl executable`,
```
Comment (JayXTQ) [inline docs/troubleshooting/common-issues/reticulating-splines.md]:
```suggestion
2. Kill The `curl` executable,
```

**Review by JayXTQ — CHANGES_REQUESTED** [review]
the fix for the first one is just a / behind getting-started. The second one it should be different, check my requests

**Review by JayXTQ — APPROVED** [review]
Xyueta changed after I made my review. The code works now and I'm fine with pushing

File: docs/troubleshooting/common-issues/reticulating-splines.md (hunk 1 above)
Comment (PencilNavigator) [reply docs/troubleshooting/common-issues/reticulating-splines.md]: xyueta was faster then me lol

**Pull request merged**